HPXMLtoOpenStudio/resources/EPvalidator.xml

```diff
@@ -726,12 +726,13 @@
   <sch:pattern name='[MechanicalVentilationType=CFIS]'>
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:MechanicalVentilation/h:VentilationFans/h:VentilationFan[h:UsedForWholeBuildingVentilation="true" and h:FanType="central fan integrated supply"]'>
       <sch:assert test='count(h:AttachedToHVACDistributionSystem) = 1'>Expected 1 element(s) for xpath: AttachedToHVACDistributionSystem</sch:assert>
+      <sch:assert test='count(h:IsSharedSystem) = 0'>Expected 0 element(s) for xpath: IsSharedSystem</sch:assert>
```

Comment:
Hmm, this seems like it wouldn't allow a HPXML file where a CFIS system has IsSharedSystem=false.

Instead, how about checking that the fan type is not CFIS inside [MechanicalVentilationType=Shared]? (This is a similar approach used to check that systems w/ preheating or precooling aren't exhaust only, see lines 744 and 752 for example.)

Reply:
I've fixed this in the last commit where I checked there's no issharedsystem=true for  [MechanicalVentilationFanType=CFIS], should be equivalent to checking system type inside [MechanicalVentilationType=Shared]